Jenkinsfile

```diff
@@ -8,26 +8,27 @@ pipeline {
         timestamps()
     }
     parameters {
-        booleanParam(name: 'unit', defaultValue: true, description: 'x86 unit tests')
+        booleanParam(name: 'unit_validate', defaultValue: true, description: 'x86 unit tests and vendor check')
         booleanParam(name: 'janky', defaultValue: true, description: 'x86 Build/Test')
```

Comment:
Also this is more of a knitpick but would it be useful to modify the description of `janky`.  The reason us what's the difference between the tests being ran in `unit_validate` and `janky`?

Reply:
It really should be named `integration-cli` although I have no idea what the difference now between `janky` and `experimental` is now